lms/static/scripts/frontend_apps/components/dashboard/test/AllCoursesActivity-test.js

```diff
@@ -154,50 +164,74 @@ describe('AllCoursesActivity', () => {
     });
   });
 
-  [12, 35, 1, 500].forEach(id => {
-    it('builds expected href for row confirmation', () => {
-      const wrapper = createComponent();
-      const href = wrapper
+  context('when building row confirmation links', () => {
+    function getLinkForID(wrapper, id) {
+      return wrapper
         .find('OrderableActivityTable')
         .props()
         .navigateOnConfirmRow({ id });
+    }
 
-      assert.equal(href, `/courses/${id}`);
+    [12, 35, 1, 500].forEach(id => {
```

Comment:
```suggestion
    [12, 35, 1, 500].forEach(courseId => {
```

Make it a little more obvious what these numbers are.